lxd/db/cluster/stmt.go

```diff
@@ -67,10 +67,17 @@ func StmtString(code int) (string, error) {
 // Warning: These triggers are applied separately to the schema update mechanism. Changes to these triggers (especially their names)
 // may require a patch.
 func applyTriggers(ctx context.Context, tx *sql.Tx) error {
-	for _, triggerStmt := range entityDeletionTriggers {
-		_, err := tx.ExecContext(ctx, triggerStmt)
-		if err != nil {
-			return err
+	for _, entityTypeInfo := range entityTypes {
+		if entityTypeInfo.onDeleteTriggerName() != "" {
```

Comment:
I suppose we should store the name and trigger sql into local vars here and check they are both non-empty, and if only one is non-empty we should return an error.

Reply:
We could also have just a `onDeleteTriggerSQL()` function signature return 2 strings, the trigger name and the sql, rather than 2 different functions. Helps to keep the concepts together.

Reply:
Done.